axelrod/tests/strategies/test_defector.py

```diff
@@ -44,9 +44,18 @@ def test_strategy(self):
         # Starts by defecting.
         self.first_play_test(D)
         self.second_play_test(D, D, D, D)
-        # Test if strategy tries to trick opponent.
```

Comment:
as far as i can tell, the old test never showed TrickyDefector being tricky.  the new tests include both normal and tricky behavior.